pkgs/servers/monitoring/nagios-plugins/manubulon-snmp-plugins/default.nix

```diff
@@ -0,0 +1,197 @@
+    })
+    (fetchDebianPatch {
+      inherit version;
+      patch = "19_check_snmp_int_remove_unneeded_my";
+      debianRevision = "4";
+      hash = "sha256-C3Bh94aRRq94kg94un3rEaCb5W7z3gNiD4n8FFel9fk=";
+      pname = "nagios-snmp-plugins";
+    })
+    (fetchDebianPatch {
+      inherit version;
+      patch = "20_check_snmp_int_avaid_huge_amount_of_regex";
+      debianRevision = "4";
+      hash = "sha256-Kh3hk7xZNWHmYlDM4k6QTFdetklspsBVvIl2S9SO6AA=";
+      pname = "nagios-snmp-plugins";
+    })
+    (fetchDebianPatch {
+      inherit version;
+      debianRevision = "4";
+      patch = "21_check_snmp_load_update_fortiswitch_and_fortigate4.3";
+      hash = "sha256-D6q1ZZ0tMHcvIu8EWyx4A5w+5Tzj68JK30kL7fm679A=";
+      pname = "nagios-snmp-plugins";
+    })
+    (fetchDebianPatch {
+      inherit version;
+      debianRevision = "4";
+      patch = "22_check_snmp_storage_fix_space_btrfs";
+      hash = "sha256-fqq1UUmqzYdxEpcrelGxrhqRghaH7BWAueTBmr8czW0=";
+      pname = "nagios-snmp-plugins";
+    })
+    (fetchDebianPatch {
+      inherit version;
+      debianRevision = "4";
+      patch = "23_check_snmp_int_interface_speed";
+      hash = "sha256-dn5yzqsSfwpVimxzBKIEhX64RR0C29ORiv3F+uKUIk8=";
+      pname = "nagios-snmp-plugins";
+    })
+    (fetchDebianPatch {
+      inherit version;
+      debianRevision = "4";
+      patch = "24_tcp_udp_ipv4_ipv6";
+      hash = "sha256-W5bEGqX2bAT4t0jey6p9TBT28ckiLPxPZWcajjyI/xU=";
+      pname = "nagios-snmp-plugins";
+    })
+    (fetchDebianPatch {
+      inherit version;
+      debianRevision = "4";
+      patch = "25_check_snmp_int_add_metrik";
+      hash = "sha256-5IlnjOAb9e2SscG5fBQ7hfJXAOwZtKOGWnPuQigXrUo=";
+      pname = "nagios-snmp-plugins";
+    })
+    (fetchDebianPatch {
+      inherit version;
+      debianRevision = "4";
+      patch = "50_disable_epn";
+      hash = "sha256-Qsf/4JeLV2P+XUnHDCH6QYaRL7bdFJQSEdhpkfsZTgw=";
+      pname = "nagios-snmp-plugins";
+    })
+    (fetchDebianPatch {
+      inherit version;
+      debianRevision = "4";
+      patch = "51_fix_privacy_doc";
+      hash = "sha256-ZIYrNCZSTWSfgcR+Oe5Nuap/az3DjuD+Uil05RtXE+M=";
+      pname = "nagios-snmp-plugins";
+    })
+  ];
+
+  buildInputs = with perlPackages; [
+    CryptDES
+    CryptRijndael
+    DigestHMAC
+    DigestSHA1
+    GetoptLongDescriptive
+    NetSNMP
+    perl
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir --parents $out/bin
+    install --mode=0755 plugins/*.pl $out/bin
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    for f in $out/bin/* ; do
+      wrapProgram $f --set PERL5LIB $PERL5LIB --set LC_ALL C
+    done
+  '';
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
+      package = manubulon-snmp-plugins;
+      # Program returns status code 3
+      command = "check_snmp_int.pl --version || true";
+      version = "check_snmp_int version : ${version}";
```

Comment:
```suggestion
```

This line is not needed.
`testers.testVersion` will check that `version` is contained in the output.

Reply:
Done, thanks!